modules/bpi/bpi.syndicate.inc

```diff
@@ -923,18 +927,25 @@ function bpi_build_top_rank(&$form) {
   $rows = bpi_build_top_rank_rows($topAgenciesPush);
   $form['bpi_top_rank']['top_agency_pushes'] = array(
     '#theme' => 'table',
-    '#caption' => t('Top pushes (3 months)'),
-    '#header' => $heading,
+    '#caption' => t('Top agency contributor (3 months)'),
+    '#header' => array(
+      t('Library'),
+      t('Amount of articles'),
+    ),
     '#rows' => $rows,
     '#empty' => t('No data so far.'),
   );
 
-  $topNodesSyndicatedCreatedByMe = bpi_get_top_ranked('-3 months', BPI_TOP_RANK_AMOUNT, 'syndicate', 'node', ['999999']);
-  $rows = bpi_build_top_rank_rows($topNodesSyndicatedCreatedByMe);
+  $self_agency = variable_get('ting_agency', '');
+  $topNodesSyndicatedCreatedByMyLibrary = bpi_get_top_ranked('-3 months', BPI_TOP_RANK_AMOUNT, 'syndicate', 'node', [$self_agency]);
+  $rows = bpi_build_top_rank_rows($topNodesSyndicatedCreatedByMyLibrary, '/bpi/edit');
```

Comment:
In general we use snake_case for variable names - not camelCase.

Reply:
My bad.